Coding/fw/src/NVMAdapter.h

```diff
@@ -0,0 +1,124 @@
+/***************************************************************************************************
+  (c) NewTec GmbH 2020   -   www.newtec.de
+  $URL: https://github.com/NewTec-GmbH/esp32-can-iot $
+***************************************************************************************************/
+/**
+@addtogroup Adapters
+@{
+@file       NVMAdapter.h
+
+ESP32 NVM Adapter for Lawicel Protocol
+
+* @}
+***************************************************************************************************/
+#ifndef NVM_ADAPTER_H
+#define NVM_ADAPTER_H
+
+/* INCLUDES ***************************************************************************************/
+#include <NVMInterface.h>
+#include <Preferences.h>
+
+/* C-Interface ************************************************************************************/
+extern "C"
+{
+}
+
+/* FORWARD DECLARATIONS ***************************************************************************/
+
+class NVMAdapter : public NVMInterface
+{
+public:
+    /* CONSTANTS ******************************************************************************/
+
+    /* TYPES **********************************************************************************/
+
+    /**
+     * Default constructor.
+     */
+
+    NVMAdapter() : NVMInterface()
+    {
+    }
+
+    /**
+    * Default destructor
+    */
+
+    ~NVMAdapter()
+    {
+    }
+
+    /**
+    * Initialize Module
+    */
+    uint8_t begin()
+    {
+        nvm.begin("Startup", false);
```

Comment:
what is this begin() end () pattern ?
this "Startup" is of concern. It eats memory each time used (unless string pooling is used).